features/activity/retry_on_error/feature.ts

```diff
@@ -1,26 +1,46 @@
 import { Context } from '@temporalio/activity';
+import * as wf from '@temporalio/workflow';
```

Comment:
I think this would set the pattern for everyone to follow, so we have to decide if that's what we want. Aliasing to "wf" what we want? Or maybe non-abbreviated "workflow" instead (we can change the workflow function name)? Would we rather import the parts of workflow we need even though we're in a shared file here?

Reply:
I find this most convenient and have been using this pattern for a while.
It's just my personal preference. I prefer `wf` over `workflow` because it's less typing and makes shorter lines.
Also would rather have all the utilities available without needing to add an import every time I discover something new's missing.